plugins/Vestige/Vestige.cpp

```diff
@@ -429,8 +429,36 @@ bool VestigeInstrument::handleMidiEvent( const MidiEvent& event, const TimePos&
 	return true;
 }
 
+bool VestigeInstrument::presetChangeSupported()
+{
+	return true;
+}
 
-
+void VestigeInstrument::changePreset(int bank, unsigned int preset)
+{
+	if (m_plugin == nullptr)
+	{
+		return;
+	}
```

Comment:
Short if means we can squash it to a single line :)
```suggestion
	if (!m_plugin) { return; }
```

Reply:
I thought that this style of coding is not allowed, but it seems that the coding standard document has changed. Updated to a one-liner.